readthedocs/projects/signals.py

```diff
@@ -34,5 +35,12 @@ def handle_project_import(sender, **kwargs):
             if service is not None:
                 if service.setup_webhook(project):
                     messages.success(request, _('Webhook activated'))
+                    _set = True
                 else:
                     messages.error(request, _('Webhook configuration failed'))
+    if not _set:
+        messages.error(
+            request,
+            _('No accounts available to set webhook on. '
+                'Please connect either a GitHub or BitBucket account.')
```

Comment:
Not sure if there is a strong reason to have it be in a signal. It does seem like you should be able to accomplish this all here though, with access to the user and project. But yes, should also catch the case of manual import and toss a warning there.